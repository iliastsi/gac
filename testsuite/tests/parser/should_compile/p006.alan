hello() : proc
    i : int;
{
 i = 0;
 i = 42;
 i = 1284;
 i = 00200;
 i = -1337;
}
