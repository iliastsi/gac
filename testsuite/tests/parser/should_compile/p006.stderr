==================== Parser ====================
hello() : proc
  i : int;
{ -- hello
  i = 0;
  i = 42;
  i = 1284;
  i = 200;
  i = -1337;
} -- hello


